src/tools.jl

```diff
@@ -15,4 +15,11 @@ Returns the thread id of the `n`th Julia thread in the `:default` threadpool.
     end
 end
 
+"""
+    taskid() :: UInt
+
+Return a `UInt` identifier for the current running [Task](https://docs.julialang.org/en/v1/base/parallel/#Core.Task). This identifier will 
```

Comment:
Seems the sentence terminated early.

"This identifier will be not be collision free"? Will be valid while the task is alive?

Reply:
oops, thanks!

Reply:
Completed sentence here: https://github.com/JuliaFolds2/OhMyThreads.jl/pull/14/commits/e30ff2a85071bf6474c7d4eee68c1f21e1984598